.github/workflows/ci.yml

```diff
@@ -17,9 +17,11 @@ jobs:
         run: mv st_syntax_tests/syntax_tests ./
       - name: Cleanup syntax_tests dir
         run: rm -R st_syntax_tests*
+      - name: Cleanup git dir
+        run: rm -R .git*
```

Comment:
Instead of removing the git folder, I would rather explicitly unselect it in the `find` call later.